docs/wfc3tools/wf3ccd.rst

```diff
@@ -26,16 +31,28 @@ executed, after which the switch will be set to COMPLETE in the corresponding
 output files. See `Section 3.4.2 of the WFC3 Data Handbook <https://hst-docs.stsci.edu/wfc3dhb>`_ for more information.
 
 
-Displaying output from wf3ccd in a Jupyter Notebook
-===================================================
+Running `wf3ccd` from a Python Terminal
+=======================================
+
+.. code-block:: shell
+
+    from wfc3tools import wf3ccd
+    wf3ccd(filename)
+
+
+Displaying output from `wf3ccd` in a Jupyter Notebook
+-----------------------------------------------------
 
 When calling `wf3ccd` from a Jupyter notebook, informational text output from the underlying `wf3ccd.e` program will be passed through `print` as the calibration runs by default, and show up in the user's cell. This behavior can be customized by passing your own function as the `log_func` keyword argument to `wf3ccd`. As output is read from the underlying program, the `wf3ccd` Python wrapper will call `log_func` with the contents of each line. (`print` is an obvious choice for a log function, but this also provides a way to connect `wf3ccd` to the Python logging system by passing the `logging.debug` function or similar.)
```

Comment:
function or similar). (parentheses before period)

Reply:
The parentheses were in the correct place, but I removed them entirely.

Reply:
Addressed in commit f5f479e.